src/main/java/com/zendesk/maxwell/row/RowMap.java

```diff
@@ -231,9 +231,10 @@ public String toJSON(MaxwellOutputConfig outputConfig) throws IOException {
 
 		if ( outputConfig.includesBinlogPosition )
 			g.writeStringField("position",
-				nextPosition.getGtidStr() != null
-					? nextPosition.getGtidStr()
-					: this.nextPosition.getFile() + ":" + this.nextPosition.getOffset());
+				nextPosition.getGtid() != null
+				? nextPosition.getGtid()
+				: this.nextPosition.getFile() + ":" + this.nextPosition.getOffset());
```

Comment:
ok, but let's still not re-use the `position` field.  If gtid mode is enabled we should be writing two keys, `position` and `gtid`, if not we should just write `position.